### DISCUSSION TITLE
在meatMask上能正常使用但是在imtoken上前端和合约交互无法进入合约方法

### DISCUSSION THREAD

**Discussion by EdisonLee123** [consenlabs/webview#16] [Q&A]
我们的dapp在meatMask上能正常使用但是在imtoken上前端和合约交互无法进入合约方法，我排查了下 用web3.js 语法例如
const Fun = new web3.eth.Contract(合约abi, 合约地址Addrs);实列化合约方法不成功，还有 const gasPrice = web3.eth.getGasPrice();
用web3.js获取gasPrice如果加了await例如const gasPrice = web3.eth.getGasPrice(); 就无法获取gasPrice ，

**Comment by unix** [comment]
> 如果加了await例如const gasPrice = web3.eth.getGasPrice(); 就无法获取gasPrice 

能详细描述一下调用的内容吗，另外如果认为是 Bug，请在 [在线示例](https://codesandbox.io/) 中创建一个最小可复现示例以帮助我们定位此问题。
